src/views/Guide.vue

```diff
@@ -29,20 +29,29 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     -->
 
     <h2 id="task-and-job-states">
-      Task And Job States
+      Understanding Tasks &amp; Jobs
     </h2>
-    <!-- TODO document difference between tasks and jobs -->
+
+    <p>
+      <b>Tasks</b> represent single units of activity in a Cylc workflow.<br/>
+      <b>Jobs</b> are real processes submitted by Cylc to perform a task's activity.<br/>
+      One task can have multiple jobs due to automatic retries or manual triggering.
+    </p>
+
```

Comment:
Sorry, this TODO was in the wrong place, could we move this into the "Task &amp; Job States" card.

Reply:
I wasn't following the TODO so much as putting the general description of tasks and jobs at the top of this (currently the only, but whatever) section, because it applies to both the following cards (i.e., including the modifiers one).  Alternatively, we could move modifiers into the main task/job card?

Reply:
I think this is clarification for the task-job separation card?

The idea was for the cards to be self-contained topics. Part of the reason for this is that we could re-use these elsewhere in the GUI e.g. startup-tips.

Reply:
I'm not really against the cards being entirely self-contained topics, but if so we shouldn't have a heading that applies to multiple cards which is currently the case on master as well as here:

> Task And Job States
> [card: Task Vs Job States]
> [card: Special Task States]

If we have cards categorized under headings like this, then a short topic description under the category heading would be fine.

If not, then I guess we should remove the heading as well.

Clearly you prefer the second option. But then it seems a bit odd to have task state modifiers in a separate card from the one that describes what task states are in the first place.

Reply:
OK I've moved the "section heading" content into the "Tasks and Jobs" card, and reframed the content of the "modifiers" card so that it doesn't seem quite so much like it should be inside the other one.  I'll post a screen shot...